docs/dev_guide_release/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _dev_guide:
+
+*****************
+Developer's Guide
+*****************
+
+Please go <here> for our up to date developer’s guide.
```

Comment:
How is here dealt with?

Reply:
<here> should be linked with the updated dev_guide version (most probably by an admin).

Reply:
I am not sure I follow.

Reply:
This should be the same link as it is now.

Reply:
Got it. I'll link that.